djangocms_versioning/models.py

```diff
@@ -1,51 +1,42 @@
-from django.core.exceptions import ImproperlyConfigured
-from django.db import connections, models
-from django.db.models import Max, Q
-from django.utils.timezone import localtime
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from django.db.models import Q
 
 
-class BaseVersionQuerySet(models.QuerySet):
+class VersionQuerySet(models.QuerySet):
 
     def for_grouper(self, grouper, extra_filters=None):
         """Returns all `Version`s for specified grouper object.
 
         Additional filters can be passed via extra_filters, for example
         if version model has a FK to content object that is translatable,
-        passing `Q(content__language='en')` will return only versions
+        passing `Q(language='en')` will return only versions
```

Comment:
How come just language would work now? We haven't changed the version model to include language, right?